robottelo/ui/sync.py

```diff
@@ -216,6 +216,18 @@ def enable_rh_repos(self, repos_tree):
                             poll_frequency=2,
                         )
 
+    def sync_noversion_rh_repos(self, prd, repos):
+        """Syncs RedHat repositories which do not have version.
+
+        Selects Red Hat non version Repos to Synchronize from UI.
+
+        :param repos: Repositories to sync which is a list.
+        :return: Returns True if the sync was successful.
+        :rtype: bool
+
+        """
+        return self.sync_custom_repos(prd, repos)
```

Comment:
This is done to avoid confusion, as to why custom repos are being used for syncing rh_repos.
Doc Strings does explain things above, thought will just clarify this here once again.